src/pantry.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <header class='main-header'>
       <div class="wrapper">
-        <img class="hamburger-menu" src="./images/hamburger-icon.png" alt="hamburger icon serving as menu button">
+        <img class="hamburger-menu hidden" src="./images/hamburger-icon.png" alt="hamburger icon serving as menu button">
```

Comment:
Thanks for adding those `.hidden` back in!  Thanks for your hard work on getting this done!